.github/workflows/actions.yml

```diff
@@ -0,0 +1,72 @@
+name: ci
+
+on:
+  push:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Once we use mill-vcs-version plugin, we should always use `fetch-depth: 0` with `actions/checkout`.